config/action.d/iptables-allports.conf

```diff
@@ -42,6 +53,7 @@ actioncheck = <iptables> -n -L <chain> | grep -q 'f2b-<name>[ \t]'
 # Values:  CMD
 #
 actionban = <iptables> -I f2b-<name> 1 -s <ip> -j <blocktype>
+actionban6 = <ip6tables> -I f2b-<name>6 1 -s <ip> -j <blocktype6>
```

Comment:
Honestly, I don't see the point in blocking anything else than the effective IP address.
It's the same as with IPv4 where also just hosts are blocked; not networks. This behavior I only know from the army where a group is being punished for failures of individuals.
Even though in IPv6 a host can have multiple IPv6 address, especially with Privacy Extensions enabled, it's very unlikely that it will change it's outbound IPv6 address faster than the ban will expire.
So again, I can't really join this opinion.